<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-6D8D7QH" failures="0" tests="4" name="Marah.Admin.TestCases.LoginPageTest" time="0.039" errors="1" timestamp="2021-07-24T02:35:47 EET" skipped="3">
  <testcase classname="Marah.Admin.TestCases.LoginPageTest" name="go_to_login_page" time="0.039">
    <error message="Cannot invoke &quot;org.openqa.selenium.WebDriver.navigate()&quot; because &quot;Marah.pages.LoginPage.driver&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "org.openqa.selenium.WebDriver.navigate()" because "Marah.pages.LoginPage.driver" is null
at Marah.pages.LoginPage.go_to_login_page(LoginPage.java:37)
at Marah.Admin.TestCases.LoginPageTest.go_to_login_page(LoginPageTest.java:23)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:176)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- go_to_login_page -->
  <system-out/>
  <testcase classname="Marah.Admin.TestCases.LoginPageTest" name="login_with_empty_data" time="0.000">
    <skipped/>
  </testcase> <!-- login_with_empty_data -->
  <system-out/>
  <testcase classname="Marah.Admin.TestCases.LoginPageTest" name="login_with_invalid_credentials" time="0.000">
    <skipped/>
  </testcase> <!-- login_with_invalid_credentials -->
  <system-out/>
  <testcase classname="Marah.Admin.TestCases.LoginPageTest" name="login_with_valid_credentials" time="0.000">
    <skipped/>
  </testcase> <!-- login_with_valid_credentials -->
  <system-out/>
</testsuite> <!-- Marah.Admin.TestCases.LoginPageTest -->
